packages/relay-service/tests/relay-service.test.js

```diff
@@ -93,4 +93,51 @@ describe('Relay service', () => {
       expect(result.length).toBeGreaterThanOrEqual(1);
     });
   });
+
+  describe('resolveDidcommMessage', () => {
+    let didCommMessage;
+    let payload;
+
+    beforeAll(async () => {
+      payload = {test: 'test'};
+      didCommMessage = await didcomm.encrypt({
+        recipientDids: [BOB_KEY_PAIR_DOC.controller],
+        type: 'test',
+        senderDid: ALICE_KEY_PAIR_DOC.controller,
+        payload,
+      });
+    });
+
+    it('expect to resolve JSON message', async () => {
+      const result = await RelayService.resolveDidcommMessage({
+        message: {
+          to: BOB_KEY_PAIR_DOC.controller,
+          msg: toBase64(didCommMessage),
+        },
+        keyPairDocs: [BOB_KEY_PAIR_DOC],
+      });
+
+      expect(result.payload).toStrictEqual({test: 'test'});
+    });
+
+    it('expect to handle URL', async () => {
+      // TODO: mock axios.get to return didComm message
```

Comment:
when are we coming back for this?

Reply:
sorry, it was work in progress. I've pushed the final commit now

Reply:
added the axios mock, and the ability to resolve messages from URLs